sdk/core/azure-core/src/main/java/com/azure/core/implementation/logging/DefaultLogger.java

```diff
@@ -0,0 +1,147 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.core.implementation.logging;
+
+import com.azure.core.util.logging.LogLevel;
+import java.io.PrintStream;
+import java.text.SimpleDateFormat;
+import java.util.Date;
```

Comment:
At lease do not using 'java.util.Date'.  https://programminghints.com/2017/05/still-using-java-util-date-dont/